### DISCUSSION TITLE
Mac OS 9.2 disapeared?

### DISCUSSION THREAD

**Discussion by steveparry1** [utmapp/UTM#6532] [macOS]
Why does the Mac OS 9.2 system not show up the UTM 'gallery' any more? I downloaded and used it just two months ago!

**Comment by knight-of-ni** [comment]
I just ran into this myself.

**Comment by JohnGossman78** [comment]
From my experience, having used OS7-9 on PPC in other implementations of Qemu, a recent update of UTM has broken the install of 9.2, and it's not being addressed.  I can't find anything covering the issues, just mine and another discussion.  I don't understand why UTM doesn't just let you do your set up then manually set your own flags.  Qemu only requires about 4 flags to install and run OS9 successfully.

**Reply by ghost** [reply]
I have the same problem: the installation used to work with the previous version.
Is there already an issue for this? I couldn't find any.

**Comment by johnkgibson** [comment]
I am using UTM 4.6.5 and have been utilizing various UTM configurations for different macOS versions, based on Adespoton's UTM configurations. I have posted the link to his repository below. 

https://github.com/adespoton/utmconfigs

**Comment by ghost** [comment]
Weird. They removed it from the gallery, but you can still download it from [their repo](https://github.com/utmapp/vm-downloads/releases/tag/mac-os-9.2.1).
